Tests/UnitTests/SubscriberAttributes/__Snapshots__/BackendSubscriberAttributesTests/iOS12-testPostReceiptWithSubscriberAttributesPassesErrorIfStatusCodeIsNotSuccess.1.json

```diff
@@ -24,6 +24,6 @@
       "observer_mode" : false
     },
     "method" : "POST",
-    "url" : "https://api.revenuecat.com/v1/receipts"
+    "url" : "https:\/api.revenuecat.com\/v1\/receipts"
```

Comment:
Huh, weird.

Reply:
It's not weird: #1563
We have better formatting for iOS 13+, but that's not available on iOS 12.